rune/libenclave/attestation/challenger.go

```diff
@@ -2,26 +2,26 @@ package attestation // import "github.com/inclavare-containers/rune/libenclave/a
 
 import (
 	"fmt"
+	"github.com/inclavare-containers/rune/libenclave/attestation/registration"
+	_ "github.com/inclavare-containers/rune/libenclave/attestation/sgx/challenger" // for the registration of sgxEpidChallenger
```

Comment:
I think the comment of this line should change to `// for the registration of sgx Challengers`